### PR TITLE
fix(typo): fix a typo in Patir Mystery 5

### LINKED ISSUE #10849: Typo in Remnant mission in the Patir system
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I'm on a Remnant mission in Builder Settlement in Patir, the black-hole next to the Graveyard.
After landing on the Ka'sei asteroid with Dusk, we are entering the ancient Builder structure.
at the end of one of the paragraphs, it reads:
'(...) The ring itself is too small for you to pass through it standing upright, but the low gravity of the asteroid lets you almost fly though it without much trouble. (...)'
Despite English not being my first language, I understand that it should read '(...) lets you almost fly THROUGH it without much trouble.(...)'

Thank you for your amazing work.

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Expected Behavior

It's just a typo

### Screenshots

_No response_

### Link to save file

_No response_

### Operating System

Android

### Game Source

Other (please describe)

### Game Version

v0.10.10-46

### Additional Information

Game downloaded via F-Droid

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Typo fix**

Fixes #10849

## Summary
Typo fix: though -> through.

### CODE REVIEW

**Comment by warp-core** [comment]
Why have you removed "almost"?

File: data/kahet/kahet missions.txt [hunk 1]
```diff
@@ -622,7 +622,7 @@ mission "Ka'het: Patir Mystery 5"
...
+			`	"This," Dusk says as he presses a switch in the middle of the panel, "is what we were looking for." Soon after, the door in front of you opens, revealing a small room with another door at the opposite end. You follow Dusk inside, where he takes out the scanner again to look for another control panel. Once that is open, another switch allows him to close the door. Only the lights on your helmet keep you from being submerged in complete darkness. Looking around, you notice that the architecture is distinctly alien, with rounded angles on every side of the room and buttresses on either side of the doors. The ring itself is too small for you to pass through it standing upright, but the low gravity of the asteroid lets you fly through it without much trouble. "We now have to wait for the room to pressurize," he tells you.`
```
Comment (TheGiraffe3) [inline data/kahet/kahet missions.txt:625]: It just... doesn't make sense in this context.
Maybe "much" could be removed instead, but I think this is best.

Comment (warp-core) [reply data/kahet/kahet missions.txt:625]: I'm not really sure what doesn't make sense about:
"the low gravity of the asteroid lets you almost fly through [the ring] without much trouble."

Comment (TheGiraffe3) [reply data/kahet/kahet missions.txt:625]: Maybe it's just me.
```suggestion
			`	"This," Dusk says as he presses a switch in the middle of the panel, "is what we were looking for." Soon after, the door in front of you opens, revealing a small room with another door at the opposite end. You follow Dusk inside, where he takes out the scanner again to look for another control panel. Once that is open, another switch allows him to close the door. Only the lights on your helmet keep you from being submerged in complete darkness. Looking around, you notice that the architecture is distinctly alien, with rounded angles on every side of the room and buttresses on either side of the doors. The ring itself is too small for you to pass through it standing upright, but the low gravity of the asteroid lets you almost fly through it without much trouble. "We now have to wait for the room to pressurize," he tells you.`
```

**Pull request merged**